package.json

```diff
@@ -66,13 +66,16 @@
     "classnames": "2.2.5",
     "coveralls": "2.13.1",
     "css-loader": "0.28.4",
+    "enzyme": "2.9.1",
+    "enzyme-to-json": "1.5.1",
     "eslint": "4.2.0",
     "eslint-loader": "1.9.0",
     "eslint-plugin-react": "7.1.0",
     "gh-pages": "1.0.0",
     "husky": "0.14.3",
     "inquirer": "3.2.0",
     "jest": "19.0.2",
+    "jest-enzyme": "3.4.0",
```

Comment:
ok. not sure if I explicitly added that. so I should just be able to add `enzyme` to the project, no need for jest-enzyme you're saying? I'll try and remove.